exec/rpc/src/main/java/org/apache/drill/exec/rpc/RpcBus.java

```diff
@@ -159,19 +159,15 @@ public ChannelClosedHandler(C clientConnection, Channel channel) {
     @Override
     public void operationComplete(ChannelFuture future) throws Exception {
       String msg;
-      if(local!=null) {
+      if(local != null) {
         msg = String.format("Channel closed %s <--> %s.", local, remote);
       }else{
         msg = String.format("Channel closed %s <--> %s.", future.channel().localAddress(), future.channel().remoteAddress());
       }
 
-      if (RpcBus.this.isClient()) {
-        if(local != null) {
-          logger.info(String.format(msg));
-        }
-      } else {
-        queue.channelClosed(new ChannelClosedException(msg));
-      }
+      logger.info(msg); // should we leave this at info level ?
+
+      queue.channelClosed(new ChannelClosedException(msg));
```

Comment:
Makes sense that this problem shows up in UserServer <--> UserClient comm.; not just [DRILL-3763](https://issues.apache.org/jira/browse/DRILL-3763), I think [DRILL-3823](https://issues.apache.org/jira/browse/DRILL-3823), [DRILL-3833](https://issues.apache.org/jira/browse/DRILL-3833), [DRILL-4170](https://issues.apache.org/jira/browse/DRILL-4170) and [DRILL-4298](https://issues.apache.org/jira/browse/DRILL-4298) will also be resolved :)